src/calendar/week.tsx

```diff
@@ -114,53 +123,52 @@ export default class Week extends Component<Props, {}> {
       onDayClick,
       onDisabledDayClick,
       onDayMouseEnter,
+      getISODate,
+      getWeekDayFormatted,
       blockClassName,
       activeMonth,
       selectedMax,
       selectedMin,
       weekStartsOn,
-      renderDay
+      renderWeekDay
     } = this.props
     const start = startOfWeek(date, { weekStartsOn })
     const end = endOfWeek(date, { weekStartsOn })
 
     return eachDay(start, end).map(day => {
-      const date = format(day, 'YYYY-MM-DD')
+      const date = getISODate(day)
       const isSelectable = this._dateSelectable(day)
       const isDisabled = this._dateDisabled(date)
       const isWorkDay = !isWeekend(date)
       const isCurrentMonth = isSameMonth(date, activeMonth)
       const isNextMonth = !isCurrentMonth && isAfter(date, activeMonth)
 
-      return (
-        <Day
-          blockClassName={blockClassName}
-          customRender={renderDay}
-          date={date}
-          handleOnClick={
-            isSelectable
-              ? onDayClick
-              : isDisabled
-              ? onDisabledDayClick
-              : undefined
-          }
-          handleOnEnter={isSelectable ? onDayMouseEnter : undefined}
-          isCurrentMonth={isCurrentMonth}
-          isDisabled={isDisabled}
-          isHighlighted={this._dateHighlighted(day)}
-          isMonthNext={isNextMonth}
-          isMonthPrev={!isCurrentMonth && !isNextMonth}
-          isNonSelectable={!isSelectable}
-          isSelectable={isSelectable}
-          isSelected={this._dateSelected(day)}
-          isSelectionEnd={Boolean(selectedMax && isSameDay(selectedMax, day))}
-          isSelectionStart={Boolean(selectedMin && isSameDay(selectedMin, day))}
-          isToday={isSameDay(today, day)}
-          isWeekend={!isWorkDay}
-          isWorkday={isWorkDay}
-          key={date}
-        />
-      )
+      return renderWeekDay({
+        ISODate: getISODate(day),
+        blockClassName,
+        date: day,
+        getWeekDayFormatted,
+        handleOnClick: isSelectable
+          ? onDayClick
+          : isDisabled
+          ? onDisabledDayClick
+          : () => {},
```

Comment:
Maybe extract it into `const noop = () => {}`?

Reply:
fine to me